examples/tree-view/containers/Node.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Component } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions'
+
+class Node extends Component {
+  constructor(props) {
+    super(props)
+    this.handleIncrementClick = this.handleIncrementClick.bind(this)
+    this.handleAddChildClick = this.handleAddChildClick.bind(this)
+  }
+
+  handleIncrementClick() {
+    const { increment, id } = this.props
+    increment(id)
+  }
+
+  handleAddChildClick(e) {
```

Comment:
We settled on only using ES6 features in examples a while ago. Mostly to stop alienating newcomers who are confused by the ecosystem already and assume Redux requires knowledge of ES2016+ proposals.